hack/local-up-cluster.sh

```diff
@@ -130,6 +130,22 @@ function test_docker {
     fi
 }
 
+function test_rkt {
+    if [[ -n "${RKT_PATH}" ]]; then
+      ${RKT_PATH} list 2> /dev/null 1> /dev/null
+      if [ "$?" != "0" ]; then
+        echo "Failed to successfully run 'rkt list', please verify that $RKT_PATH is the path of rkt binary."
```

Comment:
minor nit: `${RKT_PATH}` (add { }) to be consistent with the rest of this function.